dDatabase/CDClientDatabase/CDClientTables/CDPetComponentTable.cpp

```diff
@@ -0,0 +1,63 @@
+#include "CDPetComponentTable.h"
+
+namespace {
+	// Default entries for fallback
+	CDPetComponent defaultEntry {
+		.id = static_cast<unsigned int>(-1),
```

Comment:
Note to myself or aron to check what the defaults should be in the client

Reply:
defaults here are fine.  Use 0 for the default id or change the id type to int instead of uint (-1 cast to uint is weird)